compiler/ballerina-lang/src/main/java/io/ballerina/projects/BuildOptions.java

```diff
@@ -94,7 +94,7 @@ public boolean optimizeDependencyCompilation() {
      */
     @Deprecated(forRemoval = true)
```

Comment:
Shall we remove the Deprecated annotation and docs wherever applicable for this option?